atomic_reactor/plugins/post_pulp_pull.py

```diff
@@ -84,4 +84,6 @@ def run(self):
                        image_id)
         self.workflow.builder.image_id = image_id
 
-        return image_id
+        # currently we only support v1+v2 schema1
```

Comment:
the question of the various v2 schemas is addressed in osbs-3983. Right now, I'm focused on what Brew Pulp currently supports, which is v2 schema 1.

pulp_pull always runs, as far as I know, and it doesn't handle v1 vs v2 differently that I can see.

How do I query the registry to determine v1 vs v2?